jenkins_build.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Required Parameters
+#
+# The following parameters are set by Jenkins if you initiate the job
+# in the Jenkins project. If you wish to run this script in the
+# command line, feed the following parameters to the script.
+#
+# WORKSPACE: the absolute path of the directory assigned to the build
+#            as a workspace.
+# JOB_BASE_NAME: the last segment of $WORKSPACE, such as "foo" for
+#                "/bar/foo".
+
+set -ex
+
+check_var() {
+    if [ -z "$1" ]; then
+        echo ERROR: undefined variable
+        exit 1
+    fi
+}
+
+# Check if all required parameters are defined.
+echo Checking WORKSPACE;check_var $WORKSPACE;echo OK
+echo Checking JOB_BASE_NAME;check_var $JOB_BASE_NAME;echo OK
+
+# The name of the repo this job will checkout
+REPO_NAME=zenoss-prodbin
+# The branch of the repo this job will use
+if [ -z "${BRANCH}" ]; then BRANCH=support/5.2.x; fi
```

Comment:
If this is the only change relative to the version on the `develop` branch, I would change the code on the develop branch to something like:
```
echo Checking BRANCH;check_var $BRANCH;echo OK
```
And I would add a Jenkins job parameter to each of the jenkins jobs, and specify a default value for that parameter as `develop` or `support/5.2.x` depending on which branch the respective job builds.

Then I would cherry-pick the script from the `develop` branch to the `support/5.2.x` branch.

The advantages of doing it this way are:
* The build logic for branches remains as close as possible
* When we need to create a new branch in the future (e.g. support/5.3.x), we don't have to change the build script; we only have to change the default arguments on the new Jenkins jobs.